frontend/src/components/Pages/Faq/Faq.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import './Faq.css';
+import Accordion from '../../common/Accordian/Accordian';
+
+function Faq() {
+  return (
+    <div>
+      <Accordion
+        title="How can we be part of your foundation"
+        content="Please Whatsapp us at +91 7827552596"
+      />
+      <Accordion
+        title="Is it an established NGO?"
+        content="Yes, it's establshed NGO registered under society ACT."
+      />
+      <Accordion
+        title="Where does it primarily work?"
+        content="Delhi NCR , Bhilwara , Patna , Gurgaon , Sonipat and Chattisgarh"
+      />
+      <Accordion
+        title="How long has the organization been working within the community?"
+        content="On 19 May 2019 we did our first event. We are working in community from last 7 month"
+      />
+      <Accordion
+        title="What are the incentives if any?"
+        content="Internship letter , Certificate and Appreciation letter."
```

Comment:
remove hard coded data and save in some file.

Reply:
Sure